engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-subscriptions.controller.js

```diff
@@ -13,25 +13,28 @@
  *   Provides the functionality for the content host details action pane.
  */
 angular.module('Bastion.content-hosts').controller('ContentHostSubscriptionsController',
-    ['$scope', '$location', 'translate', 'Nutupane', 'CurrentOrganization', 'Subscription', 'ContentHost', 'SubscriptionsHelper',
-    function ($scope, $location, translate, Nutupane, CurrentOrganization, Subscription, ContentHost, SubscriptionsHelper) {
-
-        var params = {
-            'id': $scope.$stateParams.contentHostId,
-            'organization_id': CurrentOrganization,
-            'search': $location.search().search || "",
-            'sort_order': 'ASC'
-        };
+    ['$scope', '$location', 'translate', 'Nutupane', 'CurrentOrganization', 'Subscription', 'ContentHost', 'HostSubscription', 'SubscriptionsHelper',
+    function ($scope, $location, translate, Nutupane, CurrentOrganization, Subscription, ContentHost, HostSubscription, SubscriptionsHelper) {
+
+        $scope.contentHost.$promise.then(function() {
+            var params = {
+                'id': $scope.contentHost.host.id,
+                'organization_id': CurrentOrganization,
+                'search': $location.search().search || "",
+                'sort_order': 'ASC'
+            };
 
-        $scope.contentNutupane = new Nutupane(ContentHost, params, 'subscriptions');
-        $scope.detailsTable = $scope.contentNutupane.table;
```

Comment:
As discovered in #11507, this will result in `numSelected`, `getSelected()` and any other method defined directly on `alch-table` (or it's child directives) to be overridden and non-existent on the resulting table.